mtools/mplotqueries/mplotqueries.py

```diff
@@ -69,6 +70,8 @@ def __init__(self):
                                           "all existing overlays. Use "
                                           "'--overlay reset' to clear all "
                                           "overlays."))
+        # SERVER-41349 - --dns flag to plot slow DNS Resolutions; Can be grouped by connector
```

Comment:
as per Enrique comment, same here about connector. Remove the SERVER ticket reference.

Reply:
seconded